backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Services/CommitFilesService.cs

```diff
@@ -0,0 +1,207 @@
+                saved.Add(file);
+            }
+        }
+        foreach (var selected in selectedItems.Functions)
+        {
+            var item = staged.DbFunctionDetails?.Details.FirstOrDefault(x => x.Name == selected);
+            if (item != null)
+            {
+                var blobContent = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(item));
+                var file = new CommitFileDto
+                {
+                    BlobId = $"{item.Schema}-{item.Name}".ToLower(),
+                    FileName = item.Name,
+                    FileType = Shared.Enums.DatabaseItemType.Function
+                };
+                await SaveToBlob(file, selectedItems.CommitId, blobContent);
+                saved.Add(file);
+            }
+        }
+        foreach (var selected in selectedItems.Tables)
+        {
+            var item = staged.DbTableStructures?.FirstOrDefault(x => x.Name == selected);
+            if (item != null)
+            {
+                var blobContent = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(item));
+                var file = new CommitFileDto
+                {
+                    BlobId = $"{item.Schema}-{item.Name}".ToLower(),
+                    FileName = item.Name,
+                    FileType = Shared.Enums.DatabaseItemType.Table
+                };
+                await SaveToBlob(file, selectedItems.CommitId, blobContent);
+                saved.Add(file);
+            }
+        }
+        foreach (var selected in selectedItems.Constraints)
+        {
+            var item = staged.DbConstraints.FirstOrDefault(x => x.Name == selected);
+            if (item != null)
+            {
+                var blobContent = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(item));
+                var file = new CommitFileDto
+                {
+                    BlobId = $"{item.Schema}-{item.Name}".ToLower(),
+                    FileName = item.Name,
+                    FileType = Shared.Enums.DatabaseItemType.Constraint
+                };
+                await SaveToBlob(file, selectedItems.CommitId, blobContent);
+                saved.Add(file);
+            }
+        }
+        foreach (var selected in selectedItems.Views)
+        {
+            var item = staged.DbViewsDetails.Details.FirstOrDefault(x => x.Name == selected);
+            if (item != null)
+            {
+                var blobContent = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(item));
+                var file = new CommitFileDto
+                {
+                    BlobId = $"{item.Schema}-{item.Name}".ToLower(),
+                    FileName = item.Name,
+                    FileType = Shared.Enums.DatabaseItemType.View
+                };
+                await SaveToBlob(file, selectedItems.CommitId, blobContent);
+                saved.Add(file);
+            }
+        }
+        foreach (var selected in selectedItems.Types)
```

Comment:
do we still need it?

```
foreach (var selected in selectedItems.Types)
        {
            // TODO
        }
```

Reply:
Types are in progress for now(#155), it will be extended after #155 merge